content/en/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function.md

```diff
@@ -173,9 +173,20 @@ If you are collecting logs from a CloudWatch log group, configure the trigger to
 For Terraform users, you can provision and manage your triggers using the [aws_cloudwatch_log_subscription_filter][1] resource. See sample code below.
 
 ```conf
+data "aws_cloudwatch_log_group" "some_log_group" {
+  name = "/some/log/group"
+}
+
+resource "aws_lambda_permission" "lambda_permission" {
+  action        = "lambda:InvokeFunction"
+  function_name = "datadog-forwarder" # this is the default but may be different in your case
+  principal     = "logs.amazonaws.com" # or logs.amazonaws.com.cn for china
```

Comment:
```suggestion
  principal     = "logs.amazonaws.com" # or logs.amazonaws.com.cn for China*
```

Reply:
In addition to the edit above, we should include the following disclaimer when mentioning the use of the AWS China regions:

`*All use of Datadog Services in (or in connection with environments within) mainland China is subject to the disclaimer published in the [Restricted Service Locations](https://www.datadoghq.com/legal/restricted-service-locations/) section on our website.`

Example [here](https://docs.datadoghq.com/integrations/amazon_web_services/#manual). It looks like putting the disclaimer after the code block, but before the end of the tab would be a good location for it. I'm not able to add a suggestion there though. Let me know if you have any questions.